test/unit/ingestors/dans_ingestor_test.rb

```diff
@@ -5,6 +5,11 @@ class DansIngestorTest < ActiveSupport::TestCase
     @user = users(:regular_user)
     @content_provider = content_providers(:another_portal_provider)
     mock_ingestions
+    mock_timezone('Asia/Damascus') # System time zone should not affect test result
```

Comment:
wouldn't you want to randomize these with factorybot?

Reply:
Yes, it would probably be a good idea to pick a random timezone.

I'll keep some fixed though, to avoid the small chance that it randomly picks working timezones and causes a false positive test pass